cmd/bnsd/app/datamigration.go

```diff
@@ -11,25 +11,29 @@ import (
 )
 
 func init() {
-	admin, err := weave.ParseAddress("cond:gov/rule/0000000000000003")
+	technicalExecutors, err := weave.ParseAddress("cond:gov/rule/0000000000000003")
+	if err != nil {
+		panic(err)
+	}
+	governingBoard, err := weave.ParseAddress("cond:gov/rule/0000000000000001")
 	if err != nil {
 		panic(err)
 	}
 
 	datamigration.MustRegister("initialize x/msgfee configuration owner", datamigration.Migration{
-		RequiredSigners: []weave.Address{admin},
+		RequiredSigners: []weave.Address{technicalExecutors, governingBoard},
```

Comment:
This is almost impossible to achieve with our tooling. Only one of those can be the signer.

Reply:
What does this mean?

Reply:
Both "cond:gov/rule/0000000000000001" and "cond:gov/rule/0000000000000003" can never sign a transaction in the strict sense of "signing", since no keypair exist for those commitees. However, they can authorize a message execution via a governance proposal. One governance proposal is created for one commitee only. So either "cond:gov/rule/0000000000000001" or "cond:gov/rule/0000000000000003" would authorize the message.

The implementation of `RequiredSigners` requires authorization from all listed addresses:

```go
	for _, s := range m.RequiredSigners {
		if !h.auth.HasAddress(ctx, s) {
			return nil, nil, errors.Wrap(errors.ErrUnauthorized, "missing signature")
		}
	}
```

As a consequence, we will never get the required authorization the way the PR is written at the moment. Thus the migration cannot be executed.

ps.: I suggest renaming `RequiredSigners` to `RequiredAuthorizers`.

Reply:
`governingBoard` triggers the migration.

Reply:
done